### PR TITLE
 Make CLI invocation shorter

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: presto/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN yum install -y tar util-linux
 RUN curl -SL https://s3.us-east-2.amazonaws.com/starburstdata/releases/0.188-e.0.1/presto_server_pkg.0.188-e.0.1.tar.gz \
       | tar xz \
       && rpm -i presto_server_pkg.*/*rpm \
```
Comment (kokosing) [inline presto/Dockerfile]: add `/` before `presto_server_pkg`

File: presto/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ RUN yum install -y tar util-linux
...
-      && mv $(find -type f -name 'presto-cli*-executable.jar') presto-cli.jar \
+      && mv -v /presto_server_pkg.*/presto-cli*-executable.jar /presto-cli \
       && rm -rf presto_server_pkg.*
```
Comment (kokosing) [inline presto/Dockerfile]: and add slash here, so it will be consistent in all the file ;)

File: presto/Dockerfile [hunk 3]
```diff
@@ -3,8 +3,7 @@ MAINTAINER Grzegorz Kokosiński <grzegorz.kokosinski@starburstdata.com>
 
 RUN yum install -y tar util-linux
 
-RUN curl -SL https://s3.us-east-2.amazonaws.com/starburstdata/releases/0.188-e.0.1/presto_server_pkg.0.188-e.0.1.tar.gz \
-      | tar xz \
+RUN curl -SL https://s3.us-east-2.amazonaws.com/starburstdata/releases/0.188-e.0.1/presto_server_pkg.0.188-e.0.1.tar.gz | tar xz \
```
Comment (kokosing) [inline presto/Dockerfile]: Please undo, lets have a single command per line

**Comment by kokosing** [comment]
Ah you already merged that... then nevermind

**Comment by findepi** [comment]
sorry :/

**Pull request merged**